manuscript/Figure4.md

```diff
@@ -0,0 +1,465 @@
+# Figure3 - RandomForest prediction of cogScores from DEM, Taxonomic and Functional Profiles
+
+```julia
+using Chain
+using CSV
+using Statistics
+using Random
+using DataFrames
+using StableRNGs
+using MLJ
+using CairoMakie
+using DecisionTree
+using JLD2
+using Resonance
+using Distributions
+using Clustering, Distances
+ml_rng = StableRNG(0)
+```
+
+## Defining relevant functions
+
+### Calculating fitness-weighted importances
+```julia
+function calculate_fitness(train_cor::Vector{Float64}, test_cor::Vector{Float64})
```

Comment:
Are these the same as for figure 3? All the more reason to stick them inside the package